### DISCUSSION TITLE
Δεν λειτουργεί πλέον το online judge

### DISCUSSION THREAD

**Discussion by PanagiotesTec** [pdp-archive/pdp-archive.github.io#237] [General]
Παρακαλώ αν μπορείτε δείτε το, δεν κάνουν compile τα αρχεία!!

**Comment by Dim131** [comment]
Ευχαριστούμε και συγγνώμη για αυτό, τώρα πρέπει να είναι εντάξει. 

Αν ξαναγίνει, ενημέρωσέ μας!
